sources/packages/api/src/utilities/date-utils.ts

```diff
@@ -50,19 +50,16 @@ export const dateString = (date: string | Date): string => {
 };
 
 /**
- * find the date difference
- * @param firstDate first date.
- * @param lastDate Last date.
+ * Difference in days between endDate and startDate (endDate-startDate).
+ * @param endDate end date.
+ * @param startDate start date.
  * @returns the date difference in days.
  */
 export const dateDifference = (
-  firstDate: string | Date,
-  lastDate: string | Date,
+  endDate: string | Date,
```

Comment:
I think the order should be reversed, right?  `firstDate` is `startDate` and `lastDate` is `endDate`

Reply:
I would say that both are valid.
If you would be doing a subtraction of 2 dates, I would say that the most natural way would be doing BiggerDate-SmallerDate, which would be EndDate-StartDate, right?
For instance, it is the same that `dayjs` follows.
![image](https://user-images.githubusercontent.com/61259237/180273668-07fcd57b-1d48-40f5-96bf-48544664aebf.png)

Does it make sense? We can talk about it 😉

Reply:
okay..that good